spec/rails_admin/support/csv_converter_spec.rb

```diff
@@ -32,10 +32,23 @@
 
     subject { RailsAdmin::CSVConverter.new(objects, schema).to_csv(encoding_to: encoding) }
 
+    context 'when encoding FROM MySQL utf8mb4' do
```

Comment:
Please mark this as `active_record: true`, since it is causing Mongoid builds to fail.